### PR TITLE
new

### PR DESCRIPTION
**Pull request by veronikatimkina**

### CODE REVIEW

File: src/main/java/pack/ClassThree.java [hunk 1]
```diff
@@ -1,4 +1,5 @@
 package pack;
 
+@Deprecated
 public class ClassThree {
```
Comment (testaccount120) [inline src/main/java/pack/ClassThree.java:4]: 1

File: src/main/java/pack/ClassTwo.java [hunk 2]
```diff
@@ -1,4 +1,5 @@
 package pack;
 
 public class ClassTwo {
+    //efref
```
Comment (testaccount120) [inline src/main/java/pack/ClassTwo.java:4]: 2

File: src/main/java/pack/ClassOne.java [hunk 3]
```diff
@@ -1,4 +1,5 @@
 package pack;
 
 public class ClassOne {
+    public void foo(){}
```
Comment (testaccount120) [inline src/main/java/pack/ClassOne.java:4]: 3

File: src/main/java/pack/ClassThree.java (hunk 1 above)
Comment (veronikatimkina) [inline src/main/java/pack/ClassThree.java:4]: test shortcut1

File: src/main/java/pack/ClassThree.java [hunk 4]
```diff
@@ -1,4 +1,5 @@
 package pack;
 
+@Deprecated
```
Comment (veronikatimkina) [inline src/main/java/pack/ClassThree.java:3]: test single comment in distraction mode with shortcut

File: src/main/java/pack/ClassThree.java (hunk 1 above)
Comment (veronikatimkina) [reply src/main/java/pack/ClassThree.java:4]: ggg

File: src/main/java/pack/ClassThree.java (hunk 4 above)
Comment (festralik) [inline src/main/java/pack/ClassThree.java:3]: single comment

Comment (festralik) [reply src/main/java/pack/ClassThree.java:3]: reply

**Pull request merged**